### DISCUSSION TITLE
OpenTelemetry-Collector

### DISCUSSION THREAD

**Discussion by davideduma** [open-telemetry/opentelemetry-java-instrumentation#7660] [Q&A]
Hey everyone.

I'm learning to use OpenTelemetry. 
Currently i have one problem with opentelemetry-collector, not recognize or not recept information from agent

Already the component have port 4317 but this problem continue.
Already i have active this flags: "-Dotel.exporter.otlp.endpoint=http://localhost:4317" and "-Dotel.metrics.exporter=otlp -Dotel.traces.exporter=otlp", "-Dotel.metrics.exporter=otlp -Dotel.traces.exporter=otlp"
And my Agent no generate any problem on console.

Command used:
java -javaagent:opentelemetry-javaagent.jar -Dotel.metrics.exporter=otlp -Dotel.traces.exporter=otlp -Dotel.resource.attributes=service.name=hello-app,service.version=1.0 -Dotel_traces_sampler=always_on -Dotel_imr_export_interval=1000 -Dotel_metric_export_interval=1000 -Dotel.exporter.otlp.endpoint=http://localhost:4317 -Dotel.traces.exporter=zipkin -jar hello-app.jar

Agent and microservice:
![image](https://user-images.githubusercontent.com/86481341/214929741-a710a103-5be6-4615-a5c3-f7f0fba374a5.png)

Collector it's not working, does not perform the traces:
![image](https://user-images.githubusercontent.com/86481341/214928673-b0108fec-e775-4e98-8f87-557e305f63fe.png)

On Docker:
![image](https://user-images.githubusercontent.com/86481341/214929228-4619cb50-f65a-4ffa-9176-950f4557a218.png)

Zipkin it's working:
![image](https://user-images.githubusercontent.com/86481341/214930743-96aefdf1-9d75-4eec-8d1e-f2ca7ad09b9e.png)
![image](https://user-images.githubusercontent.com/86481341/214931106-69f339b7-98f5-485e-8a0e-b8d01638b839.png)

Help me

Thanks!!

**Comment by mateuszrzeszutek** [comment]
Hey @davideduma ,

Just to make sure: did you generate some traffic on your app (e.g. call an endpoint) before checking if the collector received tracing? The javaagent only generates spans when something actually happens in the app; whereas metrics (in particular, JVM metrics) are being sent somewhat constantly.
Sorry if this question sounds stupid, I just want to cross out the easiest scenario of a collector not receiving span data.
